server/src/main/java/org/elasticsearch/index/IndexService.java

```diff
@@ -829,17 +829,19 @@ private void sync(final Consumer<IndexShard> sync, final String source) {
     }
 
     abstract static class BaseAsyncTask extends AbstractAsyncTask {
+
         protected final IndexService indexService;
 
-        BaseAsyncTask(IndexService indexService, TimeValue interval) {
+        BaseAsyncTask(final IndexService indexService, final TimeValue interval) {
             super(indexService.logger, indexService.threadPool, interval, true);
             this.indexService = indexService;
             rescheduleIfNecessary();
         }
 
         protected boolean mustReschedule() {
```

Comment:
can you add the `@Override` annotation to this method?

Reply:
Sure, thanks